config/application.rb

```diff
@@ -16,6 +16,7 @@
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
+ActiveSupport::Deprecation.behavior = :stderr
```

Comment:
Will this give the correct logging output? 
```suggestion
ActiveSupport::Deprecation.behavior = :log
```

Reply:
ah yeah, good call. This is already covered in with `config.active_support.deprecation` in the environment files, so I removed it.